lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -125,25 +136,34 @@ defmodule ExUnit.Filters do
       {:excluded, "due to foo filter"}
 
   """
-  @spec eval(t, t, map, [ExUnit.Test.t()]) :: :ok | {:error, binary}
+  @spec eval(t, t, map, [ExUnit.Test.t()]) ::
+          :ok | {:excluded, String.t()} | {:skipped, String.t()}
   def eval(include, exclude, tags, collection) when is_map(tags) do
-    skip? = not Enum.any?(include, &has_tag(&1, %{skip: true}, collection))
+    excluded = Enum.find_value(exclude, &has_tag(&1, tags, collection))
+    excluded? = not (!excluded)
```

Comment:
```suggestion
    excluded? = excluded != nil
```

Reply:
what about `false`?

Reply:
Can it be false? It seems it either returns a tag or nil.

Reply:
`@tag key: false`

Reply:
I haven't checked this code in a while, but [this line](https://github.com/elixir-lang/elixir/blob/ef24e78632362355bae4e53831d15057bb4885c1/lib/ex_unit/lib/ex_unit/filters.ex#L161) makes me believe `excluded` is the name of the tag (key) and not the value.

Reply:
I don't think `@tag nil` or `@tag false` wuld make sense anyway. So let's go ahead and change it.

Reply:
It was already fixed.